Rules/CTF/Scripts/CTF_PickSpawn.as

```diff
@@ -95,7 +95,7 @@ void onTick(CRules@ this)
 //hook after the change has been decided
 void onPlayerChangedTeam(CRules@ this, CPlayer@ player, u8 oldteam, u8 newteam)
 {
-	if (player !is null && player.isMyPlayer() && this.isMatchRunning())
+	if (player !is null && player.isMyPlayer() && this.isMatchRunning() && this.gamemode_name != "Sandbox")
```

Comment:
Would it not be better to just remove CTF_PickSpawn from sandbox.cfg?

Reply:
I guess so. Although it's been a year, it should be easy to test this. Might do it soon.